cpp/arcticdb/async/task_scheduler.hpp

```diff
@@ -102,8 +104,7 @@ struct SchedulerWrapper : public SchedulerType {
 };
 
 inline int64_t get_cgroup_value(const std::string& cgroup_file) {
-    const auto path = std::filesystem::path{fmt::format("/sys/fs/cgroup/{}",cgroup_file)};
-    if(std::filesystem::exists(path)){
+    if(const auto path = std::filesystem::path{fmt::format("/sys/fs/cgroup/{}",cgroup_file)}; std::filesystem::exists(path)){
```

Comment:
`std::filesystem::path` has an `operator/` which should be used for concatenating paths instead of manual formatting.